src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -159,6 +159,16 @@ public String getRankMessage(int qnNumber, int responseNumber) {
         return element.getText();
     }
     
+    public void submitWithoutConfirmationEmail() {
+        WebElement sendEmailChecbox = browser.driver.findElement(By.name(Const.ParamsNames.SEND_SUBMISSION_EMAIL));
+        WebElement submitButton = browser.driver.findElement(By.id("response_submit_button"));
+        if (sendEmailChecbox.isSelected()) {
+            click(sendEmailChecbox);
+        }
+        click(submitButton);
+        waitForPageToLoad();
```

Comment:
Why not reuse `clickSubmitButton()` instead of repeating the contents of that function?